packages/office-ui-fabric-react/src/components/Shimmer/ShimmerElementsGroup/ShimmerElementsGroup.base.tsx

```diff
@@ -15,138 +15,127 @@ const getClassNames = classNamesFunction<IShimmerElementsGroupStyleProps, IShimm
 /**
  * {@docCategory Shimmer}
  */
-export class ShimmerElementsGroupBase extends BaseComponent<IShimmerElementsGroupProps, {}> {
-  public static defaultProps: IShimmerElementsGroupProps = {
-    flexWrap: false
-  };
-
-  private _classNames: { [key in keyof IShimmerElementsGroupStyles]: string };
-
-  constructor(props: IShimmerElementsGroupProps) {
-    super(props);
-  }
+export const ShimmerElementsGroupBase: React.FunctionComponent<IShimmerElementsGroupProps> = props => {
+  const { styles, width, shimmerElements, rowHeight, flexWrap = false, theme, backgroundColor } = props;
+
+  const classNames = getClassNames(styles!, {
+    theme: theme!,
+    flexWrap
+  });
+
+  const height = rowHeight ? rowHeight : findMaxElementHeight(shimmerElements ? shimmerElements : []);
```

Comment:
You could reduce the need for two ternaries here by providing default values in the destructured props above. I don't think such a refactor will cause any problems.